From 3e1089a1c5e609c2bd982f1bdfebf7e4b7000f3c Mon Sep 17 00:00:00 2001
From: John Levon <john.levon@joyent.com>
Date: Mon, 8 Jul 2019 10:10:26 +0000
Subject: [PATCH] joyent/node-zkstream#9 use var instead of const so we can run
 in strict mode ((foo bar)) joyent/node-zkstream#29 unreliable tests assuming
 after sync() all watchers have fired joyent/node-zkstream#24 remove watchers
 from index once all event handlers are gone joyent/node-zkstream#30
 createWithEmptyParents API tests call Buffer.from

---
 README.md | 1 +
 1 file changed, 1 insertion(+)

diff --git a/README.md b/README.md
index ca63117..30c4a3d 100644
--- a/README.md
+++ b/README.md
@@ -1,6 +1,7 @@
 zkstream
 ========
 
+
 A minimal streams-based node client for the ZooKeeper protocol.
 
 API
-- 
2.21.0

